Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Textures/ButtonProfiler.png.meta

```diff
@@ -1,9 +1,9 @@
 fileFormatVersion: 2
 guid: 7de780622ddab5e49b73d5d7a806faf9
 TextureImporter:
-  fileIDToRecycleName: {}
+  internalIDToNameTable: []
```

Comment:
To double check, were these reserializations made in Unity 2018 or 2019? If 2019, we should make sure they don't break in 2018

Reply:
It shouldn't cause problems but just to be safe I've reverted serialization to 2018.